### DISCUSSION TITLE
配置太复杂，求一个高效配置，让v2raya链接响应快，冲冲冲

### DISCUSSION THREAD

**Discussion by newyorkthink** [v2rayA/v2rayA#858] [Q&A]
### 版本、安装方式、系统

1. 你在使用什么版本的v2rayA

2. 你通过什么方式安装v2rayA

3. 你所使用的操作系统

### v2rayA/v2ray运行日志：

 ```bash
 # daemon mode
 journalctl -u v2raya | tail -n 50
 sudo tail -n 50 /var/log/v2raya/v2raya.log
 
 # docker mode
 docker logs v2raya
 ```
 或者粘贴来自 Web 页面的日志

```shell
# 此处贴入运行日志



```

### v2rayA设置以及相关截图
贴入v2rayA设置，以及问题相关的截图。

**Comment by mzz2017** [comment]
图片：
![Screenshot_20230320_183951](https://user-images.githubusercontent.com/30586220/226316168-d824322f-effc-4eb6-85c1-e456d2b6e33f.png)
这样就行了

**Reply by newyorkthink** [reply]
牛逼

**Comment by aceslup** [comment]
感觉太容易崩了，日志也没什么特别报错提醒。有时候同个设置，临时关停节点，再启动就报core异常，喊看日志，日志内容 `signal: killed` 就是 `unexpected exiting: check the log for more information`，我都怀疑我看的是不是日志
